src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/all.asset.types.osx.deps.json

```diff
@@ -18,15 +18,15 @@
           "all.asset.types.dll": {}
         }
       },
-      "libuv/1.9.0": {
+      "Libuv/1.9.0": {
```

Comment:
These casing changes are caused from using the new DependencyModel version that doesn't have all the "ToLower" hacks in it to work around the NuGet breaking change for netcoreapp1.0.

---
In reply to: [111397750](https://github.com/dotnet/sdk/pull/1119#discussion_r111397750) [](ancestors = 111397750)